drivers/scheduler/k8s/k8s.go

```diff
@@ -1460,7 +1460,7 @@ func (k *K8s) UpdateTasksID(ctx *scheduler.Context, id string) error {
 }
 
 // GetUpdatedSpec gets the updated spec of a K8s Object. Just `name`, and if required `namespace` must be specified on `spec` in order to GET the spec from K8s (if it exists).
-func (k *K8s) GetUpdatedSpec(spec interface{}) (interface{}, error) {
+func GetUpdatedSpec(spec interface{}) (interface{}, error) {
```

Comment:
@lsrinivas-pure Are we sure this will not created issues on the px side of things for you folks?

Reply:
I checked this. This function was created and called only by backup tests.
Moreover the receiver type is redundant here.

Reply:
Okay cool